### PR TITLE
use jsDelivr CDN, update jQuery & fancybox versions

### PR DESCRIPTION
**Pull request by tomByrer**
I'm a bit biased since I volunteer for jsDelivr :)  Here are the advantages vs `ajax.googleapis.com`
+ can [collate all JS or CSS](https://github.com/jsdelivr/jsdelivr#load-multiple-files-with-single-http-request) into single file & HTTP request (slightly better net compression also).  The combined file is cached in the POP after 5+ calls.
+ much better performance in [China](http://www.jsdelivr.com/features/network-map)

The chance of a browser's cache has `//ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js` is very slim.
Works with my site, but I don't use jQuery of anything else by FancyBox.

jsDelivr also allows [version auto-updating](https://github.com/jsdelivr/jsdelivr#version-aliasing), but I personally don't recommend that past minor updates.  (eg `jquery@2.1,fancybox@2.1` will serve jQuery 2.1.5 & FancyBox v2.1.9 if/when available.)

According to my brief tests here in Denver, speed is about the same for a website I'm working on, even though Google has a POP inside my city, & jsDelivr does not yet (will sometime).  Could be a more improvement gap if I [maxed out origin HTTP requests](http://tombyrer.github.io/slideshow-improving_load_times_with_cdns/#Browsers_HTTP_Connections) (eg had lots of pictures).

Thank you for all your hard work!

### CODE REVIEW

**Comment by tomByrer** [comment]
I just noticed Grunt downloads FancyBox.  Let me know if you want me to amend this PR to eliminate that.

**Comment by Xuanwo** [comment]
I merge the patch for font loading, and other patch will be better to be included in a `jsDelivr landscape`.

Maybe you can fork this repo or provide an option for user, and if option given, users may have more choice for CDN, such as cdnjs and so on.

BTW, jsDelivr look good, I do like your work.

**Comment by tomByrer** [comment]
> fork this repo or provide an option for user

I prefer the idea of jsDeliver CDN or origin/local hosting if you want me to add a switch.  Though I think the switch should exist in {project root}, so I will want to add a PR in hexojs/hexo.

cdnjs is good, but jsDelivr better since both use [CloudFlare](https://www.cloudflare.com/network-map/), but jsDelivr uses 3 other CDNs that are multiplexed into a [mega-CDN](http://www.jsdelivr.com/features/network-map).  jsDelivr is in countries cdnjs is not, like Russia, Turkey, Israel...  Plus has redundant POPs in many cities.
So I don't know an advantage of using another CDN

**Comment by tomByrer** [comment]
I'm beginning to think that external tools would be the most flexible & powerful solution.
https://www.npmjs.com/package/gulp-cdnizer
